AvalonStudio/AvalonStudio.Extensibility/Projects/AvalonStudioSolution.cs

```diff
@@ -194,7 +194,7 @@ public static VisualStudioSolution ConvertToSln (ISolution solution)
 
                 foreach (var project in solution.Projects)
                 {
-                    project.ResolveReferences();
+                    project.ResolveReferencesAsync();
```

Comment:
I'm guessing that this call is supposed to be `await`ed?

Reply:
well spotted, thanks for that! I knew something wierd was going on there!